### PR TITLE
Clinic database

### PR DESCRIPTION
**Pull request by guerreiropedr0**
In this pull request, we:
- Created a file named `schema_based_on_diagram.sql` where we implemented the database from this diagram.
clinic_diagram

1. Created a database `clinic`
2. Created tables `medical_histories`, `patients`, `treatments`, `invoices`, `invoice_items` with the correct data types
3. Created table `medical_info` for a many-to-many relationship
4. Added FK for the specific columns needed

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   treatment_id INT
+);
+
+BEGIN;
+ALTER TABLE invoice_items
+ADD FOREIGN KEY ( invoice_id ) REFERENCES invoices ( id );
+
+ALTER TABLE invoice_items
+ADD FOREIGN KEY ( treatment_id ) REFERENCES treatments ( id );
+COMMIT;
```
Comment (rukundoeric) [inline schema_based_on_diagram.sql:55]: - [ ] <b>REQUIRED:</b> Kindly create corresponding indexes to all FOREIGN KEYS as required by the project.

```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
CREATE INDEX ON medical_histories_has_treatments (treatment_id);

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @guerreiropedr0 && @TimmyChan99, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (guerreiropedr0) [reply schema_based_on_diagram.sql:55]: Thank you for your observation, we completely missed out on that

**Review by t-yanick — APPROVED** [review]
Hi @guerreiropedr0, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**